images/runtime/php/7.0/Dockerfile.7.0

```diff
@@ -0,0 +1,235 @@
+# Commit b4319e8f767f1200c9013e08baf2c34b9c84e301 from repo https://github.com/docker-library/php.git
+FROM php-run-base
```

Comment:
nit: add the comment like source for php-7.0 as it's not available in ://github.com/docker-library/php.git anymore